DataTables.Blazor/DataTable.razor

```diff
@@ -126,4 +278,81 @@
     public ValueTask DisposeAsync() => Datatables.DestroyAsync(_tableReference);
 
     private bool IsDomSourced() => string.IsNullOrWhiteSpace(SourceUrl) && Data == null;
+
+    private async Task AddEventListenersBeforeInitAsync()
+    {
+        if (OnInit.HasDelegate)
```

Comment:
This is something of a personal preference, but braces can make life easier when adding new lines and avoid silly errors like [this](https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/)

Reply:
It might also be worth covering these in a test?

Reply:
Done